Dynatrace/dynatrace-operator
# Disable CGO and use pure Go Sqlite library

## Description

[K8S-9898](https://dt-rnd.atlassian.net/browse/K8S-9898)

Disable CGO and use pure Go library for Sqlite

Build time _should_ be decreased with this change (maybe too less to notice)

![image](https://github.com/user-attachments/assets/0d967c5b-4be0-4302-abe8-d96970345244)

## How can this be tested?

- Everything should work fine
- DB exists and its populated (see pic for commands to check this)
![image](https://github.com/user-attachments/assets/d3e3e6cb-e59c-4774-9a46-2cd20e3a2c82)
